### PR TITLE
Added '.alum' selector to User Highligher

### PR DESCRIPTION
**Pull request by chewong**
#2844 
https://gyazo.com/a544c6d78b898b3ca26c95e36b6b83ea
first time contributing

### CODE REVIEW

**Comment by andytuba** [comment]
Neat!  That's the same color as the alum distinguish symbol, right? (sorry, on my phone.)

**Comment by chewong** [comment]
That's right

File: lib/templates/commentNavigator.mustache [hunk 1]
```diff
@@ -15,6 +15,7 @@
         <option name="videos">videos</option>
         <option name="popular">popular</option>
         <option name="new">new</option>
+        <option name="alum">alum</option>
```
Comment (erikdesjardins) [inline lib/templates/commentNavigator.mustache]: This template is for commentNavigator, so we don't want to add this until #3111

**Comment by erikdesjardins** [comment]
Merged in #3556, thanks!

Sorry about the delay.

**Pull request closed without merging**